lib/arcBase.js

```diff
@@ -30,8 +30,9 @@ var arc = module.exports = Base.extend({
         var chart = this.attributes.opts.chart;
         var data = this.attributes.data;
         var svg = this._make('svg',{
-            width: chart.width,
-            height: chart.height,
+            "style": "width: 100%; max-width: "
```

Comment:
just double checked the max-width is indeed not a valid property in svg. But using  ```max-width``` to make it responsive may work with a certain set of browsers (http://thatemil.com/blog/2014/04/06/intrinsic-sizing-of-svg-in-responsive-web-design/) .  May want to take a look at this to do it right. https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/preserveAspectRatio